numerics/root_finders_body.hpp

```diff
@@ -72,10 +72,9 @@ std::set<Argument> SolveQuadraticEquation(
 
   static Discriminant const discriminant_zero{};
 
-  // Use compensated summation for the discriminant because there can be
+  // Use double precision for the discriminant because there can be
   // cancellations.
-  DoublePrecision<Discriminant> discriminant(a1 * a1);
-  discriminant.Increment(-4.0 * a0 * a2);
+  DoublePrecision<Discriminant> discriminant = TwoSum(a1 * a1, -4.0 * a0 * a2);
```

Comment:
What does Higham actually do? if Higham only does compensated summation, mention the difference in the comment above.